tests/router.test.ts

```diff
@@ -1,11 +1,17 @@
 import { describe, it, expect } from "vitest";
 import { createRouter, NODE_TYPES } from "../src";
 
-export function createRoutes(paths) {
+export function createRoutes(paths: string[]) {
   return Object.fromEntries(paths.map((path) => [path, { path }]));
 }
 
-function testRouter(paths, tests?) {
+function createRoutesWithMethod(routes: string[]) {
+  return Object.fromEntries(
+    routes.map((path, i) => [path, [i & 1 ? "GET" : "POST", { path }]])
```

Comment:
It looks like an odd implementation. We could maybe extend existing `createRoutes` even with a simple parser to translate `/foo/bar.get` to `/foo/bar` + `method: GET` or support an object/array syntax (to be normalzied into normal `inserRoute`)

Reply:
Refactored to an object syntax